src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs

```diff
@@ -28,44 +28,51 @@ public class AdminIndexHandler : DatabaseRequestHandler
         [RavenAction("/databases/*/admin/indexes", "PUT", AuthorizationStatus.DatabaseAdmin, DisableOnCpuCreditsExhaustion = true)]
         public async Task Put()
         {
```

Comment:
@karmeli87 create set of issues to handle those endpoints using Processors

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-18103